packages/default/scss/stepper/_layout.scss

```diff
@@ -0,0 +1,228 @@
+            display: flex;
+            align-items: center;
+            overflow: hidden;
+        }
+        .k-step-disabled {
+            pointer-events: none;
+
+            .k-step-link { cursor: default; }
+        }
+
+
+        // Step indicator
+        .k-step-indicator {
+            @include border-radius( $stepper-indicator-border-radius );
+            margin: if($stepper-indicator-focus-offset > 0, $stepper-indicator-focus-offset, null);
+            width: $stepper-indicator-width;
+            height: $stepper-indicator-height;
+            border-width: $stepper-indicator-border-width;
+            border-style: solid;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            position: relative;
+            z-index: 1;
+            overflow: visible;
+
+            &::after {
+                @include border-radius( 100% );
+                content: "";
+                border-width: $stepper-indicator-focus-border-width;
+                border-style: solid;
+                border-color: inherit;
+                pointer-events: none;
+                display: none;
+                position: absolute;
+                top: calc( -1 * #{$stepper-indicator-focus-calc-offset} );
+                right: calc( -1 * #{$stepper-indicator-focus-calc-offset} );
+                bottom: calc( -1 * #{$stepper-indicator-focus-calc-offset} );
+                left: calc( -1 * #{$stepper-indicator-focus-calc-offset} );
+                z-index: 2;
+            }
+        }
+        .k-step-focus,
+        .k-step-link:focus {
+            .k-step-indicator::after {
+                display: block;
+            }
+        }
+
+
+        // Step label
+        .k-step-label {
+            max-width: 10em;
+            display: inline-flex;
+            flex-wrap: wrap;
+            align-items: center;
+            justify-content: center;
+            z-index: 1;
+        }
+        .k-step-label:only-child {
+            @include border-radius( ($stepper-indicator-height + 2 * $stepper-indicator-border-width) / 2 );
+            padding: $stepper-label-padding-y $stepper-label-padding-x;
+            border-width: 0;
+        }
+        .k-step-label .k-step-text {
+            max-width: calc(10em - (#{$icon-size} + #{$icon-spacing}));
+            flex-grow: 1;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+            overflow: hidden;
+        }
+        .k-step-label .k-icon { margin-left: $icon-spacing; }
+
+        .k-step-current .k-step-label,
+        .k-step-hover .k-step-label,
+        .k-step:hover .k-step-label {
+            font-weight: bold;
+        }
+
+        .k-step-label .k-step-label-optional {
+            font-style: italic;
+            flex-basis: 100%;
+        }
+
+        // Progressbar
+        .k-progressbar {
+            position: absolute;
+            pointer-events: none;
+            z-index: 0; //todo fix
```

Comment:
?

Reply:
It was -1 before and caused an issue. So it is fixed now. I just need to remove the comment.